frontend/src/main/utils/UCSBDiningCommonsMenuItemUtils.js

```diff
@@ -0,0 +1,18 @@
+import { toast } from "react-toastify";
+
+export function onDeleteSuccess(message) {
+    console.log(message);
+    toast(message);
+}
+
+export function cellToAxiosParamsDelete(cell) {
```

Comment:
If you have the cellToAxiosParamsDelete here, you need to import and use it in the UCSBDiningCommonsMenuItemTable. Also, the api url is supposed to be the camel cased one

Reply:
fixed